.gitignore

```diff
@@ -3,10 +3,10 @@ nbproject
 .idea/*
 .gitmodules
 /firebase.json
+composer.lock
```

Comment:
Why exclude this file? 

https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control